diff --git a/Engine/Source/Programs/UnrealFrontend/Private/Commands/UserInterfaceCommand.cpp b/Engine/Source/Programs/UnrealFrontend/Private/Commands/UserInterfaceCommand.cpp
index df273ea5e881..43768a67d3fa 100644
--- a/Engine/Source/Programs/UnrealFrontend/Private/Commands/UserInterfaceCommand.cpp
+++ b/Engine/Source/Programs/UnrealFrontend/Private/Commands/UserInterfaceCommand.cpp
@@ -69,8 +69,8 @@ void FUserInterfaceCommand::Run(  )
 	IModuleInterface& XCodeSourceCodeAccessModule = FModuleManager::LoadModuleChecked<IModuleInterface>( FName( "XCodeSourceCodeAccess" ) );
 	SourceCodeAccessModule.SetAccessor(FName("XCodeSourceCodeAccess"));
 #elif PLATFORM_WINDOWS
-	IModuleInterface& VisualStudioSourceCodeAccessModule = FModuleManager::LoadModuleChecked<IModuleInterface>( FName( "VisualStudioSourceCodeAccess" ) );
-	SourceCodeAccessModule.SetAccessor(FName("VisualStudioSourceCodeAccess"));
+	//IModuleInterface& VisualStudioSourceCodeAccessModule = FModuleManager::LoadModuleChecked<IModuleInterface>( FName( "VisualStudioCodeSourceCodeAccess" ) );
+	//SourceCodeAccessModule.SetAccessor(FName("VisualStudioCodeSourceCodeAccess"));
 #endif
 
 	// enter main loop
