diff --git a/Engine/Source/Runtime/D3D12RHI/Private/Windows/WindowsD3D12PipelineState.cpp b/Engine/Source/Runtime/D3D12RHI/Private/Windows/WindowsD3D12PipelineState.cpp
index bc82799f0257..bbc84be95b89 100644
--- a/Engine/Source/Runtime/D3D12RHI/Private/Windows/WindowsD3D12PipelineState.cpp
+++ b/Engine/Source/Runtime/D3D12RHI/Private/Windows/WindowsD3D12PipelineState.cpp
@@ -899,7 +899,7 @@ static void CreatePipelineStateWithExtensions(ID3D12PipelineState** PSO, FD3D12A
 					ShdExtensionDesc.uavSlot,
 					ShdExtensionDesc.registerSpace
 				);
-				check(NvStatus == NVAPI_OK);
+				//check(NvStatus == NVAPI_OK);
 
 				CreatePipelineStateWrapper(PSO, Adapter, CreationArgs);
 
@@ -909,7 +909,7 @@ static void CreatePipelineStateWithExtensions(ID3D12PipelineState** PSO, FD3D12A
 					0xFFFFFFFF,
 					0
 				);
-				check(NvStatus == NVAPI_OK);
+				//check(NvStatus == NVAPI_OK);
 
 				return;
 			}
diff --git a/Engine/Source/Runtime/RenderCore/Public/RenderUtils.h b/Engine/Source/Runtime/RenderCore/Public/RenderUtils.h
index 236031e2ddfd..0f3a747e13cf 100644
--- a/Engine/Source/Runtime/RenderCore/Public/RenderUtils.h
+++ b/Engine/Source/Runtime/RenderCore/Public/RenderUtils.h
@@ -711,7 +711,8 @@ inline bool NaniteAtomicsSupported()
 	}
 #endif
 
-	return bAtomicsSupported;
+	//return bAtomicsSupported;
+	return true;
 }
 
 inline bool DoesRuntimeSupportNanite(EShaderPlatform ShaderPlatform, bool bCheckForAtomicSupport, bool bCheckForProjectSetting)
