diff --git a/Engine/Build/BatchFiles/Build.bat b/Engine/Build/BatchFiles/Build.bat
index 9da38f1432d7..ed9caf82e20d 100755
--- a/Engine/Build/BatchFiles/Build.bat
+++ b/Engine/Build/BatchFiles/Build.bat
@@ -47,7 +47,7 @@ if not exist %UBTPath% (
 		if errorlevel 1 goto Error_UBTCompileFailed
 	) else (
 		echo Building UnrealBuildTool with dotnet...
-		dotnet build %ProjectFile% -c Development -v quiet
+		dotnet build %ProjectFile% -c Development -v quiet -maxCpuCount:1
 		if errorlevel 1 goto Error_UBTCompileFailed
 	)
 )
diff --git a/Engine/Build/BatchFiles/BuildUAT.bat b/Engine/Build/BatchFiles/BuildUAT.bat
index b67bd97a40f4..0c9179692506 100755
--- a/Engine/Build/BatchFiles/BuildUAT.bat
+++ b/Engine/Build/BatchFiles/BuildUAT.bat
@@ -89,7 +89,7 @@ call "%~dp0GetDotnetPath.bat"
 if errorlevel 1 goto Error_NoDotnetSDK
 
 echo Building AutomationTool...
-dotnet build Programs\AutomationTool\AutomationTool.csproj -c Development -v %MSBUILD_LOGLEVEL%
+dotnet build Programs\AutomationTool\AutomationTool.csproj -c Development -v %MSBUILD_LOGLEVEL% -maxCpuCount:1
 if errorlevel 1 goto Error_UATCompileFailed
 
 rem record input files - regardless of how we got here, these are now our point of reference
diff --git a/Engine/Build/BatchFiles/BuildUBT.bat b/Engine/Build/BatchFiles/BuildUBT.bat
index 01fec6f3be7e..4beb9dfbb5a2 100755
--- a/Engine/Build/BatchFiles/BuildUBT.bat
+++ b/Engine/Build/BatchFiles/BuildUBT.bat
@@ -83,7 +83,7 @@ call "%~dp0GetDotnetPath.bat"
 if errorlevel 1 goto Error_NoDotnetSDK
 
 echo Building UnrealBuildTool...
-dotnet build Programs\UnrealBuildTool\UnrealBuildTool.csproj -c Development -v %MSBUILD_LOGLEVEL%
+dotnet build Programs\UnrealBuildTool\UnrealBuildTool.csproj -c Development -v %MSBUILD_LOGLEVEL% -maxCpuCount:1
 if errorlevel 1 goto Error_UBTCompileFailed
 
 rem record input files - regardless of how we got here, these are now our point of reference
diff --git a/Engine/Source/Editor/GameProjectGeneration/Private/SProjectDialog.cpp b/Engine/Source/Editor/GameProjectGeneration/Private/SProjectDialog.cpp
index ae73874df0a2..f6ca0b9aa9f8 100644
--- a/Engine/Source/Editor/GameProjectGeneration/Private/SProjectDialog.cpp
+++ b/Engine/Source/Editor/GameProjectGeneration/Private/SProjectDialog.cpp
@@ -1752,6 +1752,7 @@ void SProjectDialog::CreateAndOpenProject()
 		else
 		{
 			OpenCodeIDE(ProjectFile);
+			OpenProject(ProjectFile);
 		}
 	}
 	else
diff --git a/Engine/Source/Runtime/Launch/Private/LaunchEngineLoop.cpp b/Engine/Source/Runtime/Launch/Private/LaunchEngineLoop.cpp
index 62471dcb8175..1a15cb1330f8 100644
--- a/Engine/Source/Runtime/Launch/Private/LaunchEngineLoop.cpp
+++ b/Engine/Source/Runtime/Launch/Private/LaunchEngineLoop.cpp
@@ -6496,6 +6496,8 @@ bool FEngineLoop::AppInit( )
 			}
 		}
 
+		bNeedCompile = false;
+
 		FEmbeddedCommunication::ForceTick(16);
 		
 		if(bNeedCompile)
diff --git a/Engine/Source/ThirdParty/Windows/VisualStudioDTE/VisualStudioDTE.Build.cs b/Engine/Source/ThirdParty/Windows/VisualStudioDTE/VisualStudioDTE.Build.cs
index 80bbb5dd1a9a..386486e8a01c 100644
--- a/Engine/Source/ThirdParty/Windows/VisualStudioDTE/VisualStudioDTE.Build.cs
+++ b/Engine/Source/ThirdParty/Windows/VisualStudioDTE/VisualStudioDTE.Build.cs
@@ -15,6 +15,8 @@ namespace UnrealBuildTool.Rules
 
 			PublicSystemIncludePaths.Add(ModuleDirectory);
 
+			PublicDefinitions.Add("WITH_VISUALSTUDIO_DTE=0");
+
 			if (Target.Platform != UnrealBuildTool.UnrealTargetPlatform.Win64 ||
 				Target.WindowsPlatform.Compiler == WindowsCompiler.Clang ||
 				Target.StaticAnalyzer == StaticAnalyzer.PVSStudio)
@@ -28,7 +30,7 @@ namespace UnrealBuildTool.Rules
 				if (TryGetDteOlbPath(out DteOlbPath))
 				{
 					TypeLibraries.Add(new TypeLibrary(DteOlbPath, "lcid(\"0\") raw_interfaces_only named_guids", "dte80a.tlh"));
-					PublicDefinitions.Add("WITH_VISUALSTUDIO_DTE=1");
+					//PublicDefinitions.Add("WITH_VISUALSTUDIO_DTE=1");
 				}
 				else
 				{
diff --git a/Engine/Source/UnrealEditor.Target.cs b/Engine/Source/UnrealEditor.Target.cs
index 5f7cb7bc4a03..90eb5efe0559 100644
--- a/Engine/Source/UnrealEditor.Target.cs
+++ b/Engine/Source/UnrealEditor.Target.cs
@@ -11,6 +11,7 @@ public class UnrealEditorTarget : TargetRules
 		IncludeOrderVersion = EngineIncludeOrderVersion.Latest;
 		BuildEnvironment = TargetBuildEnvironment.Shared;
 		bBuildAllModules = true;
+		DisablePlugins.Add("ADOSupport");
 		ExtraModuleNames.Add("UnrealGame");
 	}
 }
